### PR TITLE
Fix HttpInboundDispatchTestCase

### PR DESCRIPTION
**Pull request by prabushi**
## Description
Fix HttpInboundDispatchTestCase in transport module.

### CODE REVIEW

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/http/inbound/transport/test/HttpInboundDispatchTestCase.java [hunk 1]
```diff
@@ -37,175 +30,64 @@
  */
 
 public class HttpInboundDispatchTestCase extends ESBIntegrationTest {
-    private LogViewerClient logViewerClient = null;
+
+    private static final String requestPayload = "<soapenv:Envelope xmlns:soapenv='http://schemas.xmlsoap.org/soap/envelope/' >"
```
Comment (a5anka) [inline integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/http/inbound/transport/test/HttpInboundDispatchTestCase.java]:
```suggestion
private static final String REQUEST_PAYLOAD = "<soapenv:Envelope xmlns:soapenv='http://schemas.xmlsoap.org/soap/envelope/' >"
```

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/http/inbound/transport/test/HttpInboundDispatchTestCase.java [hunk 2]
```diff
@@ -37,175 +30,64 @@
...
+        carbonLogReader = new CarbonLogReader();
+        carbonLogReader.start();
 
     }
 
     @Test(groups = "wso2.esb", description = "Inbound HTTP Super Tenant Sequence Dispatch")
     public void inboundHttpSuperSequenceTest() throws Exception {
-        axis2Client.sendSimpleStockQuoteRequest("http://localhost:9090/", null, "WSO2");
+
+        HttpRequestUtil.doPost(new URL("http://localhost:9090/"), requestPayload, new HashMap<String, String>());
```
Comment (a5anka) [inline integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/http/inbound/transport/test/HttpInboundDispatchTestCase.java]: Shall we get the hostname and the port from a config file?

Comment (Nirothipan) [reply integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/http/inbound/transport/test/HttpInboundDispatchTestCase.java]: We have a method in ESBIntegrationTest ( super class ) to get the host name.

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/http/inbound/transport/test/HttpInboundDispatchTestCase.java (hunk 1 above)
Comment (prabushi) [reply integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/http/inbound/transport/test/HttpInboundDispatchTestCase.java]: Fixed in https://github.com/wso2/micro-integrator/pull/580

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/http/inbound/transport/test/HttpInboundDispatchTestCase.java (hunk 2 above)
Comment (prabushi) [reply integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/http/inbound/transport/test/HttpInboundDispatchTestCase.java]: Only hostname can be inferred as the post is defined in the [inbound synapse configuration](https://github.com/wso2/micro-integrator/blob/integration-tests-mediation-profile/integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/http.inbound.transport/inbound1.xml#L10).

Fixed in https://github.com/wso2/micro-integrator/pull/580

**Pull request merged**